.travis.yml

```diff
@@ -29,8 +29,9 @@ bundler_args: --without changelog development docgen guard maintenance omnibus_p
 
 matrix:
   include:
-    - rvm: 2.2
-    - rvm: 2.1
+    - rvm: 2.3.1
+    - rvm: 2.2.5
+    - rvm: 2.1.9
```

Comment:
Actually @lamont-granquist, we should probably bump the required_ruby_version when we fix up our dependencies to include 2.2-and-above gems, I'd think? Pair the change with the actual bump?